server/config/config.go

```diff
@@ -38,9 +38,9 @@ const (
 	defaultEtcdKeyPath          = ""
 	defaultEtcdCertPath         = ""
 
+	defaultEnableLimiter          bool  = true
 	defaultInitialLimiterCapacity int   = 100 * 1000
```

Comment:
If the `RateLimiter` is turned on by default, we need to carefully consider the setting of the default value for `RateLimiter`. An excessively large threshold will cause the limiter to be effectively invalid.

Reply:
By default, the rate limit threshold is set to 10000, which should be quite large. However, you can adjust the rate limit according to your specific needs and business requirements.